### ISSUE TITLE
Devise reconfirmable

### ISSUE THREAD

**Issue by b-mandelbrot** [lynndylanhurley/devise_token_auth#943]
Devise reconfirmable config didn't work with devise_auth_token.

`devise (4.2.0)
devise_token_auth (0.1.39)`

`ruby 2.3.3p222 (2016-11-21 revision 56859) [x86_64-darwin16]`

`Rails 5.0.1`

**Comment by Marinlemaignan** [comment]
yes https://github.com/lynndylanhurley/devise_token_auth/issues/1013
